topdown/http_test.go

```diff
@@ -2514,3 +2516,91 @@ var httpSendHelperRules = []string{
 		no_headers = object.remove(resp, ["headers"])
 	}`,
 }
+
+func httpSocket(t *testing.T, path string) (net.Listener, error) {
+	t.Helper()
+	sock, err := net.Listen("unix", path)
+	if err != nil {
+		return nil, err
+	}
+
+	return sock, nil
+}
```

Comment:
[nit] Now, there's very little left to this method... I suppose we could inline `net.Listen("unix", path)` in [line 2546](https://github.com/open-policy-agent/opa/pull/3667/files#diff-51c6c5bd2ad2a3fe3f9a303771fa5c82ae818d15ed0bae1560384c8b88672e92R2546) instead.